Automattic/wp-calypso
# Full Site Editing: disable gutenberg tour

#### Changes proposed in this Pull Request

* disables the tour when our plugin loads to prevent UI overlapping

#### Testing instructions

* in devtools run `delete localStorage.WP_DATA_USER_?` and replace `?` with all possible users you have there (most likely just one)
* load the new page editor and confirm there is no tour

Fixes #33626
